pkg/forth/forth.go

```diff
@@ -206,28 +257,63 @@ func String(f Forth) string {
 
 // toInt converts to int64.
 func toInt(f Forth) int64 {
+	Debug("toint %v", f.Stack())
 	c := f.Pop()
 	switch s := c.(type) {
 	case string:
+		Debug("string")
 		i, err := strconv.ParseInt(s, 0, 64)
 		if err != nil {
 			panic(err)
 		}
 		return i
 	case int64:
+		Debug("int64")
 		return s
 	default:
+		Debug("%T", c)
```

Comment:
You can put `Debug("%T", c)` before the switch and it would catch all three cases.

Reply:
done